Supernova.Enum.Generators/EnumGeneratorAttributeHelper.cs

```diff
@@ -0,0 +1,21 @@
+﻿namespace Supernova.Enum.Generators;
+
+public static class EnumGeneratorAttributeHelper
```

Comment:
Was there a special reason that you separated these features from the SourceGeneratorHelper class?

Reply:
This separation complies with the single responsibility principle and helps to keep the project clear. Based on the file name, it is now clear what this file does without having to open it.

Reply:
This separation follows the single responsibility principle and helps keep the project organized. Based on the file name, it is now clear what this file does without having to open it.